docs/compass/08-01-tutorial.md

```diff
@@ -0,0 +1,63 @@
+- [`order-service`](./assets/order-service.yaml) file that contains the service definition, deployment, and Api
+- `order-service` [API spec](./assets/order-service-api-spec.yaml)
+- [Lambda function](./assets/lambda.yaml) that calls `order-service` for orders
+- Kyma cluster with the Compass module enabled
+
+>**NOTE:** Read [this](#installation-enable-compass-in-kyma-default-kyma-installation) document to learn how to install Kyma with the Compass module enabled.
+
+## Steps
+
+### Deploy the external Application
+
+1. Log in to the Kyma Console and create a new Namespace by clicking the **Add new namespace** button.
+2. In the **Overview** tab in your Namespace, click the **Deploy new resource** button and use the file with the  [`order-service`](./assets/order-service.yaml) to connect the Application.
+3. In the **APIs** tab, copy the URL to your Application. You will use in the Compass UI in the next steps.
+
+### Register your Application in the Compass UI
+
+1. Go **Back to Namespaces** at the left top of the page and select the **Compass** tab in the left navigation panel, which navigates you to the Compass UI. Select a tenant you want to work on from the drop-down list on the top navigation panel. For the purpose of this tutorial, select the `default` tenant.
+2. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial.
+3. Navigate to the **Application** view and click **Create Application** to register your Application in Compass. For the purpose of this tutorial, name your Application `test-app`. By default, your Application and Runtime are assigned to the `DEFAULT` scenario.
+4. Click on `test-app` in the **Applications** view and add the API spec of the Order Service. To do so, click the **+** button in **API Definitions** section and fill in all the required fields. Paste the URL to your Application in the **Target URL** field. Click the **Add specification** button below and upload the `order-service` [API spec file](./assets/order-service-api-spec.yaml). In the **Credentials** tab, choose `None` as credential type from the drop-down list since, for the purpose of this tutorial, there is no need to secure the connection. Click **Create**.
+
+### Use your Application in the Kyma Console UI
+
+1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications** view. Click on `test-app` and bind it to your Namespace by clicking the  **Create Binding** button. Select the previously created Namespace from the drop-down list.
+2. Go to the **Catalog** view. See that your services are now available under the **Services** tab. Provision your instance by choosing your service and clicking the **Add once** button at the right top corner of the page.
```

Comment:
```suggestion
2. Go to the **Catalog** view. See that your services are now available under the **Services** tab. Provision the service instance by choosing your service and clicking the **Add once** button in the top-right corner of the page.
```

Reply:
I am not sure whether to use `on` or `in` here - I guess all are possible, let's just make it consistent in the doc whichever you choose